libs/features/commerce/README.md

```diff
@@ -0,0 +1,26 @@
+# Milo Commerce
+
+## Development
+
+1. Obtain corporate artifactory access token and make it accessible to npm.
+  - you may want to check `.npmrc` file for list of required npm scopes
+2. Install npm dependencies
+  - `cd libs/features/commerce`
+  - `npm i`
+3. Run Adobe Helix Cli and open Commerce test page
+  - `hlx up`
+  - `http://localhost:3000/drafts/ilyas/merch?martech=off`
```

Comment:
what is the concern to use an actual page? it might change in future, but at it exists and we all use it as a template, so for a Dev following readme it will be nice to see some prices fast